### PR TITLE
CH(Coveralls): integrate coveralls with Authors Haven

### PR DESCRIPTION
**Pull request by XROLE**
#### What does this PR do?
- Integrate coveralls with Authors Haven
#### Description of Task to be completed?
- [x] install nyc and mocha-lcov-reporter
- [x] modified test script to inlude nyc reporting
- [x] install coveralls package
- [x] add after_success script in .travis.yml file to run coverage script
- [x] add coverage script in package.json file
- [x] add nyc_output in .gitignore file
- [x] add coverage in .gitignore file
#### How should this be manually tested?
- click on [coveralls](https://github.com/andela/pippin-ah-backend/tree/CH/162727548/coveralls-integration) to see the badge 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[Finishes #162727548]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/andela/pippin-ah-backend.svg?branch=develop)](https://travis-ci.org/andela/pippin-ah-backend) [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+[![Build Status](https://travis-ci.org/andela/pippin-ah-backend.svg?branch=develop)](https://travis-ci.org/andela/pippin-ah-backend) [![Coverage Status](https://coveralls.io/repos/github/andela/pippin-ah-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/pippin-ah-backend?branch=develop) [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
```
Comment (oreoluwade) [inline README.md]: Each badge should be on its own new line.

**Comment by oreoluwade** [comment]
You need to resolve the merge conflicts

File: package.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "babel-node index.js",
     "dev:start": "babel-watch index.js",
-    "test": "mocha --require babel-polyfill --require babel-register test/*.js"
+    "test": "nyc --reporter=html --reporter=text mocha --require babel-polyfill --require babel-register test/*.js",
```
Comment (oreoluwade) [inline package.json]: I think this is all you need for the test report to show:

`nyc mocha --require babel-register`

Comment (oreoluwade) [reply package.json]: You don't need to specify the directory/directories as long as you put your tests in the `test` directory.

File: README.md (hunk 1 above)
Comment (XROLE) [reply README.md]: Ok. Doing that right away

File: package.json (hunk 2 above)
Comment (XROLE) [reply package.json]: Ok. I'll be doing that right away

File: .gitignore [hunk 3]
```diff
@@ -34,5 +34,15 @@ node_modules
...
+<<<<<<< HEAD
+<<<<<<< HEAD
 .vscode
-.env
\ No newline at end of file
+.env
+=======
+.nyc_output
+coverage
+>>>>>>> CH(Coveralls): integrate coveralls with Authors Haven
```
Comment (oreoluwade) [inline .gitignore]: Fix the merge conflicts here

Comment (XROLE) [reply .gitignore]: Conflict fixed

Comment (oreoluwade) [reply .gitignore]: It's not fixed yet. Can still see `<<<< HEAD` as well as your commit hashes.

**Review by oreoluwade — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (XROLE) [reply .gitignore]: > It's not fixed yet. Can still see `<<<< HEAD` as well as your commit hashes.

Oh, I didn't notice that of .gitignore. Fixing it asap

**Pull request merged**